app/helpers/bootstrap_flash_helper.rb

```diff
@@ -0,0 +1,30 @@
+module BootstrapFlashHelper
+  ALERT_TYPES = [:success, :info, :warning, :danger] unless const_defined?(:ALERT_TYPES)
+
+  def bootstrap_flash(options = {})
+    flash_messages = []
+    flash.each do |type, message|
+      # Skip empty messages, e.g. for devise messages set to nothing in a locale file.
```

Comment:
__[rubocop]__
* Line is too long. [86/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)